website/src/contribute/release-guide.md

```diff
@@ -1248,6 +1253,9 @@ please follow [the guide](https://help.github.com/articles/creating-a-personal-a
    delete the `.asc`, `.sha512`;
 3. Upload the new release `twine upload *` from the directory with the `.zip` and `.whl` files;
 
+[Installing twine](https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives): pip install twine
```

Comment:
This will install twine globally. This works but it may not be what people want.

suggestion -> install twine in a virtualenv.

Reply:
This is opposite to Udis suggestion to remove the virtualenv part.
Should we assume that people will know the option of running any python command in virtual env?

Reply:
I would not assume that. Not everyone works with regularly and may just run the pip command directly.

@udim what do you think?

Reply:
(replied via mail below)